Samples/VirtualCamera/VirtualCamera_ConsoleApp/readme.txt

```diff
@@ -0,0 +1,122 @@
+========================================================================
+About the Project
+========================================================================
+Basic console application that validate virtual camera source implementation, 
+register as virtual camera, remove virtual camera, validate virtual camera works 
+thru mediacapture API
+
+The mediasource used by the project as followed: 
+ VirtualCameraWin32
+ * A dll implemented with cppwinrt.
+
+ VirutlaCameraWinRT
+ * A winrt implementation of the MediaSource.
+
+    UPDATE: This implementation is NOT RECOMMENEDED, winrt component has dependency 
+    on VCRUNTIMExxx_APP.dll etc.. that is not in the system by default.
+    The mediasource will fail to load due to missing dependency
+
+Getting Started:
+    To configure this project to build, you must have access to OS repo 
+    and reference branch with the private / unpublish APIs. 
+    See step 3 in "Project Setup" to configure the project to reference
+    OS repo.
+
+WARNING:
+    Please stick with x64 only, project is not properly configure for x86!
+
+========================================================================
+Project Setup - working with internal/non-publish API
+========================================================================
+
+1. Add Wil nudget 
+2. Configure project to reference internal/unpublish interfaces
+
+    Reference the latest headers from OS repo: project -> right click -> select property -> C\C++
+        $(OSRoot)\$(ProcessorArchitecture)fre.nocil\onecore\internal\avcore\inc;
```

Comment:
> $(OSRoot)\$(ProcessorArchitecture)fre.nocil\ [](start = 7, length = 45)

as commented earlier in propsheets, we should defer to SDK

Reply:
removed